README.md

```diff
@@ -1,334 +1,26 @@
-# Helm Chart For Zabbix.
+# zabbix
```

Comment:
Hi @tomsozolins!

The README.md file was updated without use the README.md.gotmpl file. The result is more different that the expected. Do you can to try again? 

Maybe the error is in the directory mounted in make gen-docs command. Try using the helm-docs binary direct, please.